globals/strings.gd

```diff
@@ -675,6 +692,8 @@ func get_condition_text(conditions):
 				if len(required_bloom_levels) > 0:
 					bloom_str = " " + tr("(Bloom %s)") % "/".join(required_bloom_levels)
 				text += "If attached to %s%s: " % [get_names([condition["required_member_name"]])[0], bloom_str]
+			"attached_with":
+				text += "If attached with %s: " % get_names([condition["required_card_name"]])[0]
```

Comment:
please tr() this and add a line in the translations

Reply:
The whole section on that part are full of texts with no ``tr`` entries. I feel it's much better to do them in one go in a later time instead with proper translations.

Reply:
ok, that's fine